db/seeds/avatars.yml

```diff
@@ -0,0 +1,61 @@
+---
+- https://images.unsplash.com/photo-1494790108377-be9c29b29330?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzNDIyNzJ8MHwxfHNlYXJjaHwxfHxhdmF0YXJ8ZW58MHx8fHwxNjU3MTQyMTI3&ixlib=rb-1.2.1&q=80&w=1080&utm_source=railsdevs&utm_medium=referral&utm_campaign=api-credit
```

Comment:
Let's only keep the unique photo ID/key in this file and build the full URL in the seeds.

Reply:
I'm also not sure if we need all of the params, like `ixid`?

Reply:
I will remove those params and only keep the required ones. 

I would suggest keeping the full url rather than just the ID as it makes it much easier for you to check one particular images from the seed. But can do the change if you prefer

Reply:
That's a good point. But I don't it's worth the trade-off of having so much additional code in the YAML file.

As a compromise, we could expose the helper to a public class method on the helper so it can be used without much work in the console.